src/System.Resources.Reader/src/System/Resources/ResourceReader.cs

```diff
@@ -264,7 +266,7 @@ private void _ReadResources()
             //  may be sufficient, but let's plan for the future)
             //  Skip over alignment stuff.  All public .resources files
             //  should be aligned   No need to verify the byte values.
-            long pos = _stream.Position;
+            long pos = _stream.Position - _resourceStreamStart;
```

Comment:
Why is this subtraction necessary?  I'd have thought that the required alignment would be absolute rather than relative to some position in the stream.